test/savingLoading.js

```diff
@@ -7,9 +7,11 @@ const range = require('range').range
 const { Vec3 } = require('vec3')
 const assert = require('assert')
 
-const testedVersions = ['1.8', '1.13', '1.14', '1.16', '1.17', '1.18']
+const todoVersions = ['1.9', '1.10', '1.11', '1.12']
+const testedVersions = require('../').supportedVersions
 
 for (const version of testedVersions) {
+  if (todoVersions.includes(version)) continue
```

Comment:
Why are you skipping versions? Either all versions should be tested or you should revert this change.

Reply:
As I said in the previous PR all versions wasn't tested in the previous version of the code either. I just refactored it make it clear but didn't change the list of tested versions here

Reply:
Remove the line

Reply:
Or is your point that there is missing test data for these versions? If so that should be written explicitly in a comment

Reply:
ok...

Reply:
@extremeheat The point of this change is to ensure newly added versions in supported versions are tested without needing to add new versions here (you may forget to do it). This was made for convenience. Could I just rename it to `skippedVersions`? Or will it still not be clear?

Reply:
I think it's better we explicitly state what versions are tested

That's what we do in other packages and it works pretty well

It means the pattern for adding new versions is to add in tested versions then fix the tests, which makes sense